packages/react/menu/src/Menu.tsx

```diff
@@ -186,11 +188,9 @@ const MenuSubTrigger = React.forwardRef((props, forwardedRef) => {
         onMouseMove={composeEventHandlers(props.onMouseMove, () => {
           if (!props.disabled) context.onOpenChange(true);
         })}
-        onKeyDown={composeEventHandlers(props.onKeyDown, (event) => {
-          if (!props.disabled && SUB_OPEN_KEYS[context.dir].includes(event.key)) {
-            context.onKeyOpen();
-          }
-        })}
+        onSearchExclusiveKeyDown={(event) => {
```

Comment:
Since we are probably going to keep this... I was going to suggest renaming this to just `onExclusiveKeyDown` since it is only fired when we're not searching so the `Search` part in the name was a bit confusing.